docs/source/markdown/podman-network.1.md

```diff
@@ -33,7 +33,7 @@ so networks have to be created again after a backend change.
 | prune      | [podman-network-prune(1)](podman-network-prune.1.md)           | Remove all unused networks                                      |
 | reload     | [podman-network-reload(1)](podman-network-reload.1.md)         | Reload network configuration for containers                     |
 | rm         | [podman-network-rm(1)](podman-network-rm.1.md)                 | Remove one or more networks                                     |
-| update     | [podman-network-upate(1)](podman-network-update.1.md)          | Update an existing Podman network                               |
+| update     | [podman-network-update(1)](podman-network-update.1.md)          | Update an existing Podman network                               |
```

Comment:
This is something I wish I could check for in `xref-helpmsgs-manpages` but there are too many (valid) inconsistencies in subcommand names and man page paths. Thank you for catching this one. LGTM.

Reply:
Actually, we _can_ crosscheck. #17486 will prevent this sort of mistake from happening again. Thank you for the inspiration.